### DISCUSSION TITLE
Allow packed ice to melt in the nether

### DISCUSSION THREAD

**Discussion by 1ByteBit** [PurpurMC/Purpur#1615] [Feature Requests]
Or, to be a broader feature:
- Allow deciding whether packed and/or blue ice can melt, and if it only melts in the nether

**Comment by Onako2** [comment]
Sounds interesting!

**Comment by Axiumyuyu** [comment]
to make nether has possible to exists water? sounds doesn't bad.

**Reply by 1ByteBit** [reply]
No, ice already melts in the nether and doesn't release water. That is what I'm requesting for packed (and maybe blue) ice.

Allowing water to exist in the nether would have to be a separate suggestion
